### ISSUE TITLE
Width setting should define editor width in characters

### ISSUE THREAD

**Issue by raphinesse** [defunkt/Zen#82]
I have width set to the default of 80. With the default configuration this leaves me with an 80 character wide editor in zen mode. When I enable the Gutter and/or the Minimap, their width seems to be **subtracted** from the width of the editor, leaving me with a narrower editor than desired. Instead,, the editor width should stay at the exact width configured, while any extras add to the total width.

**Comment by braver** [comment]
You are correct. The scroll bar, if there is one, is also subtracted from the configured width. Fixing this properly is complicated: scroll bars may appear and disappear based on the content of the editor and the size of the viewport. Also, the width of the minimap is based on the width of the editor. I don't see a good solution without polling and race conditions. But if you do, I'm open for a PR.

**Comment by raphinesse** [comment]
I almost suspected that it was a more complicated matter. Right now, I have no time to give it a shot myself. But it would be nice if you could post a link to the relevant sections of the code that might give me or others a good jump start. TIA

**Comment by braver** [comment]
Look for the `# Set width` comment.
